lib/sqlalchemy/orm/session.py

```diff
@@ -1902,7 +1903,8 @@ def begin_nested(self) -> SessionTransaction:
 
             :ref:`pysqlite_serializable` - special workarounds required
             with the SQLite driver in order for SAVEPOINT to work
-            correctly.
+            correctly. The same holds true for the aiosqlite driver
```

Comment:
**mike bayer** (zzzeek) wrote:

"For asyncio use cases, see the section :ref:`aiosqlite_serializable`"

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4907